branch/cs/SEM5/CS305/index.md

```
@@ -0,0 +1,115 @@
+|  1  | **Mid Semester** | [📥 Open](./PYQ/Mid-Semester/) |
+|  2  | **End Semester** | [📥 Open](./PYQ/End-Semester/) |
+
+</div>
+
+---
+
+## 📤 Contributing
+
+<details>
+<summary><b>How to add resources</b></summary>
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)